Updated messaging around SDK docs (#1353)
temporalio/documentation: docs/app-dev-context/foundations.md

Some sections may be incomplete or missing for some languages.
Information may change at any time.

> If you can not find what you are looking for in the Application development guide, it could be in [older docs for SDKs](/sdks).

Reviewer: ```suggestion
If you can't find what you are looking for in the Application development guide, it could be in [older docs for SDKs](/sdks).
```
